index.html

```diff
@@ -95,84 +95,7 @@ <h2 class="mobile-work-header">My Recent </br> Works</h2>
         <h2 class="desktop-work-header">My Recent Works</h2>
     </div>
     <div class="myworks-wrapper">
```

Comment:
- I noticed the popup on the desktop version hides some text, please try to reduce the height of the popup or add a scroll so users can see the text when they scroll down. 

##### Your Design
![image](https://github.com/demesameneshoa/Portfolio/assets/58771507/7c4e45c5-2378-4701-b935-704263216301)

##### Figma design

![image](https://github.com/demesameneshoa/Portfolio/assets/58771507/8f6df915-0890-4cc8-8867-e583ffbedf93)